tests/simulation/translator/ref/Flow360_om6Wing.json

```diff
@@ -44,7 +44,7 @@
         },
         "lowMachPreconditioner": false,
         "maxForceJacUpdatePhysicalSteps": 0,
-        "typeName": "Compressible",
+        "modelType": "Compressible",
```

Comment:
why is this ref JSON changed? Where in solver preprocessing do we change this typeName to modelType?